tests/test_data/rxnrel.csv

```diff
@@ -1,12 +1,12 @@
 "rxcui1","rxaui1","stype1","rel","rxcui2","rxaui2","stype2","rela","rui","srui","sab","sl","rg"
-"1819",,"CUI","RN","1818",,"CUI","tradename_of","4716626",,"RXNORM",,
-"1819",,"CUI","RN","203840",,"CUI","tradename_of","4317610",,"RXNORM",,
+"1819",,"CUI","RN","1818",,"CUI","reformulated_to","4716626",,"RXNORM",,
+"1819",,"CUI","RN","203840",,"CUI","reformulated_to","4317610",,"RXNORM",,
```

Comment:
This change was to make the included_rels table actually detect something.

Reply:
Was this file generated/downloaded originally? Maybe add the fact that you made this edit to the generation instructions too.

Reply:
ok, i'll add a note someplace